episodes/choosing-acquisition-settings.md

```diff
@@ -0,0 +1,546 @@
+
+## Light exposure (i.e phototoxicity / photobleaching)
+
+Illuminating our samples with light is an essential part of collecting light 
+microscopy images. This being said, we have to be very careful of the 
+detrimental effects light can have on our samples, especially at high intensity 
+over long time periods. 
+
+Photobleaching is a key issue for fluorescence microscopy. If a fluorophore is 
+exposed to intense light, its structure can degrade over time. This means that 
+it will eventually stop fluorescing entirely.
+
+Another issue is phototoxicity. Light exposure can damage cells resulting in 
+unexpected changes in cell behaviour, morphology and eventually cell death.
+
+Reducing the effects of photobleaching and phototoxicity requires minimising the 
+light exposure of our samples as much as possible. For example, reducing 
+light/laser power, reducing exposure time/dwell time, imaging at lower temporal 
+resolution...
+
+## Signal to noise ratio
+
+Signal to noise ratio is a very useful measure of image quality - in general, it 
+will be easier to identify and measure features of interest in images with a 
+higher signal to noise ratio.
+
+The 'signal' is what we really want to measure - for example, for a fluorescence 
+microscopy image, this would be the light emitted by fluorophores in the sample. 
+In an ideal world the detector would perfectly measure the light intensity, but 
+this isn't really possible. The values the detector records will always be 
+affected by random fluctuations called 'noise'. This noise usually appears as a 
+random 'graininess' over the image and can make details difficult to see.
+
+![](fig/image-with-noise.png){alt="Left - the nuclei from Napari's Cells 
+(3D+2Ch) sample image. Right - same image with added gaussian noise"}
+
+To identify and measure our features of interest, the absolute size of the 
+signal and noise are less important than the ratio between them. For example, 
+imagine we are trying to distinguish between two areas with different brightness 
+- one being 1.5x brighter than the other. In a low signal setup where values are 
+small (say 20 vs 30) adding noise can make these areas very difficult to 
+separate. For example, see the diagram below which shows three histograms. The 
+left is the ideal scenario, where there is no noise, and we perfectly see one 
+value of 20 and one of 30. The middle shows the addition of some noise to each 
+value, resulting in wider distributions that overlap extensively. This means 
+that in the combined histogram of these values (right) it becomes impossible to 
+clearly distinguish these two values.
+
+![](fig/snr-comparison-low.png){alt="Diagram of a low signal-to-noise scenario. 
+Left - histogram with no noise. Middle - histogram with added noise (separate 
+histograms). Right - histogram with added noise (combined histogram)."}
+
+If we imagine a scenario with higher signal, now values of 80 and 120 (one still 
```

Comment:
Can we add some specifics here on the what the SNR was for each set of images. Left of first image, signal = 20, noise = 0, SNR = infinite. Middle image signal = 20 , noise = 10? SNR = 2. Bottom row. signal = 80, noise = 10? SNR = 8?

Reply:
I'm a little wary to put hard SNR values on these images as there are lots of different ways to calculate it (e.g. [in pete bankhead's book](https://bioimagebook.github.io/chapters/3-fluorescence/3-formation_noise/formation_noise.html#fn-3)).  Also, if we just use (pixel value / standard deviation of gaussian noise) then the SNR will be different for the two pixel values - which is perhaps a bit confusing? I've left it as-is for now, but happy to adjust if preferred.